test.py

```diff
@@ -0,0 +1,8 @@
+def calculate_sum(numbers)
```

Comment:
The function definition is missing a colon at the end, which will cause a syntax error.
```suggestion
def calculate_sum(numbers):
```